src/sql/workbench/contrib/objectExplorer/browser/media/serverTreeActions.css

```diff
@@ -39,3 +39,11 @@
 .monaco-workbench.hc-black .actions-container .action-label.server-page {
 	background-image: url('server_page_inverse.svg') !important;
 }
+
+/* styles for action labels - without these, the buttons for the Servers view will not be sized correctly */
+.monaco-pane-view .pane > .pane-header > .actions .action-label.icon,
```

Comment:
If this only affects the server view then we should probably scope this down to just the server container.

If there are other icons then I think it'd be better to make our own paneview.css file and import that as well to make it easier to find these general styles. css files should ideally ONLY contain styles that apply to a single scope.